### PR TITLE
MRG: distribution project report.

### PR DESCRIPTION
**Pull request by z357412526**
We create this place to work on distribution project report together.

### CODE REVIEW

File: doc/source/report/distribution.rst [hunk 1]
```diff
@@ -6,9 +6,14 @@ Power Laws & Universality
 Statement of Problem:
 ---------------------
 
+
 To take a closer look at the tail property of locomotion, we analyze the
 moving distance distribution for home based mice and non home based
-mice.
+Mice. Fitting those distribution using some prior knowledge and obtain
```
Comment (CenzhuoYao) [inline doc/source/report/distribution.rst]: mice should not start with capital letter. Please fixed that typo. Thanks.

File: doc/source/report/distribution.rst [hunk 2]
```diff
@@ -74,20 +131,22 @@ of the tail.
...
-- Conduct statistical test to quantitatively analysis the fitness (Pearson chi
-  square test). For testing the hypothetical distributions of a given array,
-  there are several existing commonly used methods. However, each approach has
-  their pros and cons. Following is a short overview of these testing framework.
-  We recommend that all the methods are to be tried to get a comprehensive
-  understanding of the inter-event step distributions.
-- Pearson Chi-square test
-- Fisher’s exact test
-- KS test
+- Draw the histogram for our data. For each mouse day,  observe the distribution and explore whether our distribution assumption makes sense.
```
Comment (CenzhuoYao) [inline doc/source/report/distribution.rst]: Double space before "observation".

File: doc/source/report/distribution.rst [hunk 3]
```diff
@@ -74,20 +131,22 @@ of the tail.
...
+- Estimate parameters based on MLE of powerlaw and exponential.
```
Comment (CenzhuoYao) [inline doc/source/report/distribution.rst]: MLE of "truncated" power law and "truncated" exponential. Our model is truncated, not the full model. Also, the truncated power law distribution has another name called Pareto distribution.

File: doc/source/report/distribution.rst [hunk 4]
```diff
@@ -74,20 +131,22 @@ of the tail.
...
+- Add the density function to our histogram, see the fitness of our distribution.
```
Comment (CenzhuoYao) [inline doc/source/report/distribution.rst]: Some normalization may be needed to compare histogram and density function.

**Comment by CenzhuoYao** [comment]
@jarrodmillman We have finalized our report and rebased. It is ready to merge now. Thanks!

File: doc/source/report/distribution.rst [hunk 5]
```diff
@@ -74,20 +131,24 @@ of the tail.
...
-  understanding of the inter-event step distributions.
-- Pearson Chi-square test
-- Fisher’s exact test
-- KS test
+- Draw the histogram for our data. For each mouse day, observe the distribution and explore whether our distribution assumption makes sense.
+- Estimate parameters based on MLE of truncated powerlaw, aka Pareto distribution and truncated exponential.
+- Add the density function to our histogram, see the fitness of our distribution.
+- Conduct statistical test to quantitatively analysis the fitness. For testing the hypothetical distributions of a given array, there are several existing commonly used methods:
+
+   - Kolmogorov–Smirnov test
```
Comment (doutib) [inline doc/source/report/distribution.rst]: Add mann whitney U test please, currently working on it

**Pull request merged**